app/src/controllers/chefs.ts

```diff
@@ -6,44 +6,79 @@ import { IdentityProvider } from '../components/constants';
-import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionFormExport } from '../types';
+import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionForm, ChefsSubmissionFormExport } from '../types';
 
 const controller = {
   getFormExport: async (req: Request, res: Response, next: NextFunction) => {
     try {
       const cfg = config.get('server.chefs.forms') as ChefsFormConfig;
-      let formData = new Array<ChefsSubmissionFormExport>();
 
-      // Get a JSON export of all form data merged into a single array
-      await Promise.all(
+      let exportData: Array<Partial<ChefsSubmissionForm & { formId: string }>> = await Promise.all(
         Object.values<ChefsFormConfigData>(cfg).map(async (x: ChefsFormConfigData) => {
-          const data = await chefsService.getFormExport(x.id);
-          data.forEach((d: ChefsSubmissionFormExport) => (d.form.id = x.id));
-          formData = formData.concat(data);
+          return (await chefsService.getFormExport(x.id)).map((data: ChefsSubmissionFormExport) => {
+            const financiallySupportedValues = {
+              financiallySupportedBC: isTruthy(data.isBCHousingSupported),
+              financiallySupportedIndigenous: isTruthy(data.isIndigenousHousingProviderSupported),
+              financiallySupportedNonProfit: isTruthy(data.isNonProfitSupported),
+              financiallySupportedHousingCoop: isTruthy(data.isHousingCooperativeSupported)
+            };
+
+            return {
+              formId: x.id,
+              submissionId: data.form.submissionId,
+              confirmationId: data.form.confirmationId,
+              contactEmail: data.contactEmail,
+              contactPhoneNumber: data.contactPhoneNumber,
+              contactName: `${data.contactFirstName} ${data.contactLastName}`,
+              financiallySupported: Object.values(financiallySupportedValues).includes(true),
+              ...financiallySupportedValues,
+              intakeStatus: data.form.status,
+              latitude: data.latitude,
+              longitude: data.longitude,
+              naturalDisaster: data.naturalDisasterInd,
+              projectName: data.companyNameRegistered,
+              queuePriority: data.queuePriority,
+              singleFamilyUnits: data.singleFamilyUnits ?? data.multiFamilyUnits,
+              streetAddress: data.streetAddress,
+              submittedAt: data.form.createdAt,
+              submittedBy: data.form.username
+            };
+          });
         })
-      );
+      ).then((x) => x.filter((y) => y.length).flat());
+
+      // Get a list of all submission IDs
+      const result = await chefsService.searchSubmissions({
+        submissionId: exportData.map((x) => x.submissionId as string)
+      });
+
+      // Overwrite export data with application data where possible
+      exportData = exportData.map((x: Partial<ChefsSubmissionForm & { formId: string }>) => ({
```

Comment:
Nitpick: May be better to assign this to a new named const variable instead of an overwrite; that way we can have exportData be a const, and subsequent use of this would be named like `mergedExportData` or something else instead. Usually better to steer away from object/primitive reassignments unless it is really needed; seems like this variable is only used one more time during the final filter and response.